src/App/AddEmployee/AddEmployeeController.java

```diff
@@ -76,26 +78,48 @@ public JFXTextField getEmployee_department() {
     private JFXTextField employee_designation;
     @FXML
     private JFXTextField employee_department;
+    @FXML
+    JFXComboBox<String>box_empGender = new JFXComboBox<>();
+    @FXML
+    private JFXTextField employee_id;
+
+    private void setEmployeeId() throws Exception{
+
+        Connect connect = new Connect();
+        Connection connection = connect.getConnection();
+        Statement statement = connection.createStatement();
+        String sql1 = "SELECT MAX(id) as id FROM EMPLOYEE";
+        ResultSet rs = statement.executeQuery(sql1);
+        if(rs.next()) {
+            int empid = rs.getInt("id");
+
+            employee_id.setText(Integer.toString(empid+1));
```

Comment:
We know `emid` will return a single result. But we should use the `employee_id` seText method outside if condition.